manifests/exec_restorecon.pp

```diff
@@ -0,0 +1,37 @@
+# selinux::exec_restorecon
+# 
+# A convenience wrapper around a restorecon exec
+#
+# Will execute after all other SELinux changes have been applied, but before
+# Anchor['selinux::end']
+# 
+# @param path The path to run restorecon on. Defaults to resource title.
+# @param recurse Whether restorecon should recurse. Defaults to true
+# @param refreshonly see the Exec resource
+# @param unless see the Exec resource
+# @param onlyif see the Exec resource
+#
+define selinux::exec_restorecon(
+  Stdlib::Absolutepath $path = $title,
+  Boolean $refreshonly = true,
+  Boolean $recurse = true,
+  Optional[String] $unless = undef,
+  Optional[String] $onlyif = undef,
```

Comment:
can you reformat this to allign the =?

The vim plugin junegunn/vim-easy-align can easily do that for you.